src/bundle/Resources/translations/admin_ui_admin_trash_list.en.xliff

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:jms="urn:jms:translation" version="1.2">
+  <file source-language="en" target-language="en" datatype="plaintext" original="not.available">
+    <header>
+      <tool tool-id="JMSTranslationBundle" tool-name="JMSTranslationBundle" tool-version="1.1.0-DEV"/>
+      <note>The source node in most cases contains the sample message as written by the developer. If it looks like a dot-delimitted string such as "form.label.firstname", then the developer has not provided a default message.</note>
+    </header>
+    <body>
+      <trans-unit id="d0c4047caa5bbe26ed27248a3fc325cb8b972f6b" resname="confirm">
+        <source>Restore</source>
+        <target state="new">Restore</target>
```

Comment:
Can you extract translation again? This will remove the `state="new"`, thanks to that we will have a smaller diff in the future if someone adds some string and does an extraction.